services/frontend-service/pkg/config/config.go

```diff
@@ -56,6 +56,7 @@ type ServerConfig struct {
 	ApiEnableDespiteNoAuth             bool          `default:"false" split_words:"true"`
 	IapEnabled                         bool          `default:"false" split_words:"true"`
 	GrpcMaxRecvMsgSize                 int           `default:"4" split_words:"true"`
+	EnabledRevisions                   bool          `default:"false" split_words:"true"`
```

Comment:
I'd like to stick to the pattern "FooEnabled"
```suggestion
	RevisionsEnabled                   bool          `default:"false" split_words:"true"`
```